src/umockdev.vala

```diff
@@ -1018,7 +1019,10 @@ public class Testbed: GLib.Object {
                     break;
 
                 case 'S':
-                    /* TODO: ignored for now */
+                    /* collect symlinks */
+                    if (val != null) {
+                        devnode_links += Path.build_filename(this.root_dir, "dev", val);
```

Comment:
An `S:` symlink should always have a value, otherwise the line is pointless. Let's error out for empty values (`throw UMockdev.Error.PARSE`) instead of silently ignoring it.